packages/common/src/util.ts

```diff
@@ -180,7 +180,7 @@ export function mapRecord<
  * Maps an infinite stream of type A to an infinite stream of type B
  * @param mapper: function used to map each element from type A to B.
  */
-export function mapAsyncIterable<A, B>(
+export function mapStream<A, B>(
```

Comment:
Was this just a private function? Why is it renamed?

Reply:
I needed to use the previously private `asyncIterableToList()` and thought the name too verbose, so I renamed it to `streamToList()` and made it public.

I renamed `mapAsyncIterable()` as well for consistency. Neither function was public so it's not a breaking change, although since the `streamToList()` is now public I'll add it to the changelog.